### ISSUE TITLE
Removing invitation uses wrong method

### ISSUE THREAD

**Issue by roberta-pavel** [jdalrymple/gitbeaker#3638]
**Description**

- Node.js version: 18.20.4
- Gitbeaker version: 40.4.0
- Gitbeaker release (cli, rest, core, requester-utils): core
- OS & version: Mac

Removing method for invitations uses `put` method instead of `delete`

**Steps to reproduce**

```
const gitlabService = new Gitlab({
  host: HOST,
  token: TOKEN,
});

await gitlabService.GroupInvitations.remove(someId, someEmail)
```

**Expected behaviour**

Invitation gets removed

**Actual behaviour**

Error "Bad request" because the method is not correct

**Possible fixes**

Update method for remove from `put` to `del` [here](https://github.com/jdalrymple/gitbeaker/blob/main/packages/core/src/templates/ResourceInvitations.ts)

Open pull request: https://github.com/jdalrymple/gitbeaker/pull/3639

**Checklist**

- [ x ] I have checked that this is not a duplicate issue.
- [ x ] I have read the documentation.

**Comment by roberta-pavel** [comment]
#3639

**Comment by jdalrymple** [comment]
:rocket: Issue was released in [`40.5.0`](https://github.com/jdalrymple/gitbeaker/releases/tag/40.5.0) :rocket:
